kong/client.py

```diff
@@ -76,3 +76,13 @@ def cli(self):
                 raise KongError('Kong object %s not available' % name)
             result[name] = await o.apply_json(data)
         return result
+
+    async def get_service_plugin(self, service_name, plugin_name):
```

Comment:
Yes, but I don't like repeating this everytime we need to find a plugin. Just to let it reusable

Reply:
OK, but this is not efficient.
It takes two calls to kong to get the plugin. It is also implemented in the client rather than in a component.
We could store the service ID in the environment or in the cache and implement a method in the Plugins class https://github.com/lendingblock/aio-kong/blob/master/kong/plugins.py#L29
```python
def get_for_service(self, name, service_id):
     plugins = await self.get_list(name=name, service_id=service_id)
     ....
```

and get the plugin via
```python
plugin = await cli.plugins.get_for_service('jwt', service_id)
```